firmware/hw_layer/digital_input_icu.cpp

```diff
@@ -185,21 +182,7 @@ ICUDriver * getInputCaptureDriver(const char *msg, brain_pin_e hwPin) {
 		return &ICUD3;
 	}
 #endif
-#if STM32_ICU_USE_TIM8
-	if (hwPin == GPIOC_6 ||
-		hwPin == GPIOC_7) {
-		return &ICUD8;
-	}
-#endif
-#if STM32_ICU_USE_TIM9
-	if (hwPin == GPIOA_2 ||
```

Comment:
Please keep this code. Depending on configuration it could be used.

I am fine with changing default config but that's just default